src/jdk.compiler/share/classes/com/sun/tools/javac/tree/Pretty.java

```diff
@@ -1220,7 +1240,7 @@ public void visitNewArray(JCNewArray tree) {
 
     public void visitLambda(JCLambda tree) {
         try {
-            print("(");
+            print('(');
             if (tree.paramKind == JCLambda.ParameterKind.EXPLICIT) {
                 printExprs(tree.params);
             } else {
```

Comment:
nit, just below: `String sep = "";` -> `String sep = '\0';` also `sep = ","` -> sep = ','?

Reply:
Done

Reply:
This failed the build - I reverted it.